src/gluonts/model/predictor.py

```diff
@@ -134,6 +146,46 @@ def from_inputs(cls, train_iter, **params):
         params = {**auto_params, **params}
         return cls.from_hyperparameters(**params)
 
+    def backtest(
+        self,
+        test_data: TestData,
+        metrics: Union[Collection[Metric], Dict[str, Metric]],
```

Comment:
I don't like this. There should only be one way to do it.

In my view, we don't want to rename metrics.

Reply:
Still need to adjust the type :)